src/activity-log/index.html

```diff
@@ -17,13 +17,13 @@
     {% import 'macros/category-icon.html' as category_icon %}
     {% from 'post-macros.html' import pagination as pagination with context %}
 
-    <h1>Activity Log</h1>
-    <p class="h3">
+    <h1 data-qa-hook="title">Activity Log</h1>
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/bdd4a6a69f6f1e6667bacc95befe9b21b6726a65